CHANGELOG.md

```diff
@@ -2,8 +2,8 @@
 
 # master
 
-## 0.1.13 - 2019-10-08
-- Add supports for `contacts` endpoint.
+## 1.0.0 - 2019-10-08
+- No longer supports Ruby 2.1, 2.2
 
```

Comment:
I dont understand why you removed 0.1.13 changelog

Reply:
It was a mistake, I wanted to delete 0.1.14 but it's not on this branch